modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,842 @@
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+.Query
+[source, cypher]
+----
+SHOW ALIAS `northwind` FOR DATABASE
+----
+
+.Result
+[role="queryresult]
+----
++---------------------------------------------------------------------------+
+| name        | composite | database               | location | url  | user |
++---------------------------------------------------------------------------+
+| "northwind" | NULL      | "northwind-graph-2021" | "local"  | NULL | NULL |
++---------------------------------------------------------------------------+
+----
+
+==== Set properties for local database aliases
+
+Local database aliases can also be given properties.
+These properties can then be used in queries with the xref::database-administration/composite-databases/querying-composite-databases.adoc#composite-databases-queries-built-in-functions[`graph.propertiesByName()` function].
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `northwind-2022`
+FOR DATABASE `northwind-graph-2022`
+PROPERTIES { newestNorthwind: true, index: 3 }
+----
+
+The properties are then shown in the `SHOW ALIASES FOR DATABASE YIELD ...` command.
+
+.Query
+[source, cypher]
+----
+SHOW ALIAS `northwind-2022` FOR DATABASE YIELD name, properties
+----
+
+.Result
+[role="queryresult"]
+----
++------------------------------------------------------+
+| name             | properties                        |
++------------------------------------------------------+
+| "northwind-2022" | {index: 3, newestnorthwind: TRUE} |
++------------------------------------------------------+
+----
+
+==== Create database aliases with the same name as an existing alias
+
+Adding a local database alias with the same name as an existing local or remote alias does nothing with the `IF NOT EXISTS` clause but fails without it.
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `northwind` IF NOT EXISTS FOR DATABASE `northwind-graph-2020`
+----
+
+==== Create or replace database aliases
+
+It is also possible to replace a database alias.
+The old alias may be either local or remote.
+
+.Query
+[source, cypher]
+----
+CREATE OR REPLACE ALIAS `northwind` FOR DATABASE `northwind-graph-2020`
+----
+
+This is equivalent to running the following two queries consecutively:
+
+.Query
+[source, cypher]
+----
+DROP ALIAS `northwind` IF EXISTS FOR DATABASE
+----
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `northwind` FOR DATABASE `northwind-graph-2020`
+----
+
+[[database-management-create-remote-database-alias]]
+=== Create remote database aliases
+
+A database alias can target a remote database by providing an URL and the credentials of a user on the remote Neo4j DBMS.
+See link:{neo4j-docs-base-uri}/operations-manual/{page-version}/manage-databases/remote-alias[Connecting remote databases] for the necessary configurations.
```

Comment:
since this link is to the operations manual, can it be shortened? and is the link even correct anymore or has the thing it points to moved in your restructuring?